scout/server/blueprints/cases/controllers.py

```diff
@@ -794,6 +795,40 @@ def rerun(store, mail, current_user, institute_id, case_name, sender, recipient)
         LOG.error("Cannot send rerun message: no recipient defined in config.")
 
 
+def call_rerunner(store, institute_id, case_name, metadata):
+    """Call rerunner with updated pedigree metadata."""
+    # define the data to be passed
+    payload = {"case_id": case_name, "sample_ids": [m["sample_id"] for m in metadata]}
+
+    cnf = rerunner.connection_settings
+    url = cnf.get("entrypoint")
+    if not url:
+        raise ValueError("Rerunner API entrypoint not configured")
+    auth = HTTPBasicAuth(current_user.email, cnf.get("api_key"))
+    LOG.info(f"Sending request -- {url}; params={payload}")
+    resp = requests.post(
+        url,
+        params=payload,
+        json=metadata,
+        timeout=rerunner.timeout,
+        headers={"Content-Type": "application/json"},
+        auth=auth,
+    )
+
+    if resp.status_code == 200:
```

Comment:
I changed it back to the previous code, since codefactor didn't like the if True: